clients/drcachesim/tracer/tracer.cpp

```diff
@@ -2000,6 +2000,8 @@ event_exit(void)
     drmgr_exit();
     func_trace_exit();
     drx_exit();
+    /* Avoid accumulation of option values on static-link re-attach. */
+    droption_parser_t::clear_values();
```

Comment:
> There is no control point for droption to do this automatically, so we
> add droption_parser_t::clear_values() that a client can call.

Is there a control point in DR that can call clear_values? It might be better to not leave it up to the client to do this.

Reply:
> Is there a control point in DR that can call clear_values?

I mean a point in DR outside droption, of course.

Reply:
DR should not have a dependence on droption.  If droption were a regular ext library and registered DR events, it could register for the exit event: but it's a pure-header library that is used not just by clients but by regular-program front-ends so it seems nicest and cleanest to keep it separate from DR events.